apache/zeppelin
# [WIP Zeppelin-945] Interpreter authorization

### What is this PR for?
This PR is for interpreter authorization.

### What type of PR is it?
Improvement

### Todos
* [ ] - only show the lock button if admin login.

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-945

### How should this be tested?
1. login as admin
2. then you can see the lock button on the right top as following image.
![image](https://cloud.githubusercontent.com/assets/3348133/17078720/5fdedf1e-5135-11e6-8832-b2dd2c8843d8.png)
3. you can give users permission as check on the checkbox you want.
4. run paragraph job as user and see results if job has run or not properly.

### Screenshots (if appropriate)
![image](https://cloud.githubusercontent.com/assets/3348133/17078861/c8d9b5c6-5139-11e6-98a6-588a87fbdee0.png)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes
